source/sass/patterns/molecules/figure.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.figure {
+  @include block-spacing($end: $baselinegrid-space-medium);
+  @include figure-caption-typography();
+}
+
+.figure__image {
+  @include block-spacing($end: $baselinegrid-space-extra_small);
+  @include margin(12px, inline);
+  border: 1px solid $color-text-dividers;
+}
+
+.figure__heading {
+  @include figure-caption-typography();
+  @include nospace();
```

Comment:
The `<figcaption>` does need indentation on smaller screens.

Reply:
Yes. eLife handles it at https://github.com/elifesciences/pattern-library/blob/develop/assets/sass/patterns/molecules/captioned-asset.scss#L52